index.html

```diff
@@ -1,38 +1,320 @@
+                <li>Ruby on Rails</li>
+                <li>css</li>
+                <li>JavaScript</li>
+                <li>html</li>
+              </ul>
+            </div>
+            <button class="card-button">See Project</button>
+          </div>
+        </div>
+      </section>
+    </main>
+      <section class="about-us">
+        <h1 class="about-heading">About Me</h1>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, do not hestiate to contact me.
+        </p>
+      </section>
+      <button class="card-button">Get My Resume</button>
+      <div class="footer-grid">
+        <section>
+          <div class="footer-card">
+            <div>
+              <img
+                class="footer-img"
+                src="./images/icon-languages.svg"
+                alt=""
+              />
+            </div>
+            <div>
+              <h2>Languages</h2>
+              <div class="group-tag">
+                <div class="footer-list">
+                  <h4 class="footer-tags">JavaScript</h4>
+                  <h4 class="footer-tags">Ruby</h4>
+                </div>
+                <div class="footer-list">
+                  <h4 class="footer-tags">Html</h4>
+                  <h4 class="footer-tags">CSS</h4>
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section>
+          <div class="footer-card">
+            <div>
+              <img
+                class="footer-img"
+                src="./images/icon-frameworks.svg"
+                alt=""
+              />
+            </div>
+            <h2>Frameworks</h2>
+            <div class="group-tag">
+              <div class="footer-list">
+                <h4 class="footer-tags">Bootstrap</h4>
+                <h4 class="footer-tags">Ruby on Rails</h4>
+              </div>
+              <div class="footer-list">
+                <h4 class="footer-tags">RSpec</h4>
+                <h4 class="footer-tags">Capybara</h4>
+                <h4 class="footer-tags">Selenium</h4>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section>
+          <div class="footer-card">
+            <div>
+              <img class="footer-img" src="./images/icon-skills.svg" alt="" />
+            </div>
+            <div class="group-tag">
+              <h2>Skills</h2>
+              <div class="footer-list">
+                <h4 class="footer-tags">codekit</h4>
+                <h4 class="footer-tags">Github</h4>
+                <h4 class="footer-tags">Codepen</h4>
+              </div>
+              <div class="footer-list">
+                <h4 class="footer-tags">Gitlab</h4>
+                <h4 class="footer-tags">Terminal</h4>
+              </div>
+            </div>
+          </div>
+        </section>
```

Comment:
- [ ] Section doesn't have any semantic meaning here. Kindly replace that. 
You can check [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) when to use a `section` tag
`HINT: You can use a generic div if you want here.`

Reply:
Corrected. Section tag removed and replaced with div.